pixi-multistyle-text.ts

```diff
@@ -75,6 +77,39 @@ export default class MultiStyleText extends PIXI.Text {
 		this.dirty = true;
 	}
 
+	public setTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag === "default") {
+			this.textStyles[tag] = this.assign(MultiStyleText.DEFAULT_TAG_STYLE, style);
+		} else {
+			this.textStyles[tag] = this.assign({}, style);
+		}
+
+		this._style = new PIXI.TextStyle(this.textStyles["default"]);
+		this.dirty = true;
+	}
+
+	public updateTagStyle(tag: string, style: ExtendedTextStyle): void {
+		if (tag in this.textStyles) {
+			this.assign(this.textStyles[tag], style);
```

Comment:
I believe having 2 methods to "change" a style will confuse people.

For the comments, sure :)
- You're right. Acually we could also just do `if (this.textStyles[tag])` then :)
- That way you're sure to use the proper `hasOwnProperty` function, not a potential one overriden by the object
- Actually I'd like to have Babel handle the Object.assign polyfill, and anyway I believe it's safe to use Object.assign since it's available everywhere, except IE9 maybe but does Pixi still support IE9?